config/configUtil.js

```diff
@@ -73,17 +73,20 @@ function csvToArray(str) {
  *  replicas.
  */
 function getReadReplicas(pe, replicaLabel) {
-  let replicas;
+  let replicas = [];
   if (pe[replicaLabel]) {
     const replicaList = csvToArray(pe[replicaLabel]);
-    replicas = [];
     replicaList.forEach((replica) => {
       if (pe[replica]) {
         replicas.push(pe[replica]);
       }
     });
   }
 
+  if (!replicas.length) {
+    replicas = undefined;
+  }
+
   return replicas;
```

Comment:
So here is the problem with that case
Let's assume we have only one value in REPLICAS = test and there is no env variable name test. Then it will return null array instead of undefined and that will cause an error. So for handling that case I did this. only one parameter and that is bad one.